tests/integration/DOM-test.js

```diff
@@ -184,3 +185,17 @@ test('Clicking modal overlay', function(assert) {
   });
 
 });
+
+test('Pressing escape', function(assert) {
```

Comment:
As for how to test this, I think the first step will be to see if there is a way to limit the keyDown event to the 'topmost' modal. To test, you'll probably use the [keyEvent helper](http://emberjs.com/guides/testing/test-helpers/#toc_asynchronous-helpers) and then:

```
```